pandas/tests/apply/test_frame_apply.py

```diff
@@ -858,6 +858,22 @@ def test_with_dictlike_columns_with_infer():
     tm.assert_frame_equal(result, expected)
 
 
+def test_with_dictlike_functions():
```

Comment:
We're no longer changing dictlike behavior, I think this can be removed. It should fail, but that is another (unrelated) issue.